packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -343,18 +343,20 @@ class IconButton extends StatelessWidget {
     );
     final BoxConstraints adjustedConstraints = effectiveVisualDensity.effectiveConstraints(unadjustedConstraints);
 
+    final double size = iconSize ?? iconTheme.size!;
```

Comment:
`iconTheme.size` can't be null, It get a default value from `IconThemeData.fallback`. Should I add `Default value as 24` just to be sure?

Reply:
I don't think it's required. Just make sure you add a test(s) that will fail if this occurs.

Reply:
yeah, you're right. There is a pre-existing test that already verifies this. (Default icon size when no size is specified) https://github.com/flutter/flutter/blob/d3bdbed4985ca2ab174d65a6f556bd18ac03be5e/packages/flutter/test/material/icon_button_test.dart#L30